commands/sendMsg.js

```diff
@@ -4,8 +4,19 @@ const config = require('../config.json');
 
 module.exports = {
     name: 'sendMsg',
-    description: 'send message to specific channel by id',
-    async execute(message) {
-        
+    description: 'send message to specific channel by name',
+    async execute(message,args) {
+        const manageChannels = message.member.permissions.has(Discord.Permissions.FLAGS.MANAGE_CHANNELS,Discord.Permissions.FLAGS.MANAGE_MESSAGES,Discord.Permissions.FLAGS.SEND_MESSAGES,Discord.Permissions.FLAGS.VIEW_CHANNEL);
+        const manageChEmbed = new Discord.MessageEmbed()
+            .setColor(config.COLOR_EMBED)
+        if (!manageChannels) {
+                manageChEmbed.setDescription("🚫 You don't have permission to use this command!")
+            message.reply({ embeds: [manageChEmbed] });
+        }
+        const channelName = args[1];
```

Comment:
**Replace this lines :**
```js
const channelName = args[1];
const channelTarget = message.guild.channels.cache.find(channel => channel.name === channelName)
```
**with :**
```js
const guild = bot.guilds.cache.get('881560236446601216');
const channelId = args[1].replace(/\D/g, '');
const channelTarget = guild.channels.cache.get(channelId);

Reply:
#### I edited the guild part so that it could run in any guild/server without being hard-coded and instead used the following : 
### 👇
<br>

```js
        const channelID = args[1].replace(/\D/g, '');
        const channelTarget = message.guild.channels.cache.get(channelID);
```